microsetta_public_api/server.py

```diff
@@ -111,9 +114,9 @@ def run(app):
         tdb = TestDatabase()
         tdb.start()
         atexit.register(tdb.stop)
-        app = build_app()
+        app = build_app(validate_responses=False)
         run(app)
         tdb.stop()
     else:
-        app = build_app()
+        app = build_app(validate_responses=True)
```

Comment:
maybe something like `build_app(validate_responses=SERVER_CONFIG.get('validate', False))` ?

The `use_test_database` flag is different and I never ended up using it as much as I anticipated... But as it is written you it looks like you would have to toggle the data that are available for the non-dataset routes  (via `TestDatabase) along with `validate_responses` when you toggle the `use_test_database` flag?